feature-group-definitions/motion-path.yml

```diff
@@ -0,0 +1,14 @@
+spec: https://drafts.fxtf.org/motion-1/
+caniuse: css-motion-paths
+compat_features:
+  - css.properties.offset-anchor
+  - css.properties.offset-distance
+  - css.properties.offset-path
+  - css.properties.offset-path.basic-shape
+  - css.properties.offset-path.coord-box
+  - css.properties.offset-path.path
+  - css.properties.offset-path.ray
```

Comment:
Do you want to exclude this given https://developer.mozilla.org/en-US/docs/Web/CSS/offset-path#browser_compatibility? In fact it seems like css.properties.offset-path.path is the only subentry that we should include if we want this to be "old supported stuff"...